packages/react/src/components/TooltipIcon/TooltipIcon.js

```diff
@@ -47,7 +48,12 @@ TooltipIcon.propTypes = {
   /**
    * Specify the direction of the tooltip. Can be either bottom or top.
    */
-  direction: PropTypes.oneOf(['bottom', 'top']).isRequired,
+  direction: PropTypes.oneOf(['bottom', 'top']),
+
+  /**
+   * Specify the alignment (to the trigger button) of the tooltip. Can be one of: start, center or end.
+   */
+  alignment: PropTypes.oneOf(['start', 'center', 'end']),
```

Comment:
Maybe `align` could work here too? Makes it feel a bit more like CSS (although it may be intentionally different 👍)

Reply:
OK I was on the fence on this. Updated as such.